power_grid_model_io_native_c/power_grid_model_io_native/include/power_grid_model_io_native/vnf_converter/vnf_pgm_converter.hpp

```diff
@@ -0,0 +1,112 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#pragma once
+
+#include <power_grid_model_io_native_c/power_grid_model_io_native_c/src/handle.cpp>
+
+#include <power_grid_model/common/exception.hpp>
+#include <power_grid_model_c/power_grid_model_c/src/forward_declarations.hpp>
+
+#include <iostream>
+
+#ifndef POWER_GRID_MODEL_IO_NATIVE_C_VNF_PGM_CONVERTER_H
+#define POWER_GRID_MODEL_IO_NATIVE_C_VNF_PGM_CONVERTER_H
+
+#ifdef __cplusplus
+
+class PgmVnfConverter;
+
+extern "C" {
+#endif
```

Comment:
don't use `extern "C"` in header files. It tells the compiler within a translation unit that it should be ABI-compatible (C), rather than only API-compatible (C++)